google/cloud/bigquery/routine/routine.py

```diff
@@ -329,6 +335,27 @@ def remote_function_options(self, value):
             self._PROPERTY_TO_API_FIELD["remote_function_options"]
         ] = api_repr
 
+    @property
+    def data_governance_type(self):
+        """Optional[str]: If set to ``DATA_MASKING``, the function is validated
+        and made available as a masking function.
+
+        Raises:
+            ValueError:
+                If the value is not :data:`string` or :data:`None`, or the value
+                is not in `self._DATA_GOVERNANCE_TYPE`.
+        """
+        return self._properties.get(self._PROPERTY_TO_API_FIELD["data_governance_type"])
+
+    @data_governance_type.setter
+    def data_governance_type(self, value):
+        if value is not None and value not in self._DATA_GOVERNANCE_TYPE:
```

Comment:
Generally we try to avoid this kind of setter enforcement in the client library itself, as the library may not be aware of all valid values.  Allow the methods to raise errors if there's issues (e.g at routine creation/update/etc).

Reply:
Thank you, this is really good to know! I will only check for type instead.